assets/users/components/EditUser.tsx

```diff
@@ -334,38 +364,14 @@ function EditUserComponent({
             </div>
         </div>
     );
-}
-
-EditUserComponent.propTypes = {
-    original: PropTypes.object,
-    user: PropTypes.object.isRequired,
-    onChange: PropTypes.func,
-    errors: PropTypes.object,
-    companies: PropTypes.arrayOf(PropTypes.object),
-    onSave: PropTypes.func.isRequired,
-    onResetPassword: PropTypes.func.isRequired,
-    onClose: PropTypes.func.isRequired,
-    onDelete: PropTypes.func.isRequired,
-    resendUserInvite: PropTypes.func.isRequired,
-    currentUser: PropTypes.object,
-    toolbar: PropTypes.node,
-    products: PropTypes.arrayOf(PropTypes.object),
-    hideFields: PropTypes.arrayOf(PropTypes.string),
-    allSections: PropTypes.arrayOf(PropTypes.object),
-    companySections: PropTypes.object,
-    seats: PropTypes.object,
-};
-
-EditUserComponent.defaultProps = {
-    hideFields: [],
 };
 
-const mapStateToProps = (state: any) => ({
+const mapStateToProps = (state: IReduxStateProps) => ({
```

Comment:
not sure about the typing here, shouldn't this `IReduxStateProps` define what is returned from `mapStateToProps` and not what is passed there? it could be the same but in general this seems confusing, that state should be rather defined in the reducer

Reply:
No I just typed the state object with that interface here since state's type isn't infered from anywhere. The return type of `mapStateToProps` is infered, I could add it explicitly but it won't make a difference.

Reply:
Eventually yes, we should move it there, but now it would be too time consuming to debug the whole app to get all the possible values of each state prop. I think it's a good middle ground for now